src/Symfony/Bundle/WebProfilerBundle/Tests/Controller/ProfilerControllerTest.php

```diff
@@ -137,6 +137,33 @@ public function testToolbarActionWithEmptyToken($token)
         $this->assertEquals(200, $response->getStatusCode());
     }
 
+    public function testToolbarStylesheetActionWithProfilerDisabled()
+    {
+        $urlGenerator = $this->createMock(UrlGeneratorInterface::class);
+        $twig = $this->createMock(Environment::class);
+
+        $controller = new ProfilerController($urlGenerator, null, $twig, []);
+
+        $this->expectException(NotFoundHttpException::class);
+        $this->expectExceptionMessage('The profiler must be enabled.');
+
+        $controller->toolbarStylesheetAction();
+    }
+
+    public function testToolbarStylesheetAction()
+    {
+        $urlGenerator = $this->createMock(UrlGeneratorInterface::class);
+        $twig = $this->createMock(Environment::class);
+        $profiler = $this->createMock(Profiler::class);
+
+        $controller = new ProfilerController($urlGenerator, $profiler, $twig, []);
+
+        $response = $controller->toolbarStylesheetAction();
+        $this->assertEquals(200, $response->getStatusCode());
```

Comment:
assertSame please

Reply:
✅  Updated

Just this one ? Or the strings too ?

Reply:
Strings too please